README.md

```diff
@@ -110,6 +110,7 @@ Feel free to add your own project(s) -- just fork and make a pull request! We su
 - **[badges/shields](https://github.com/badges/shields)** ([shields.io](https://shields.io/category/downloads)) - A badge generator with a badge for download totals on Modrinth projects
 - **[Gaming32/Superpack](https://github.com/Gaming32/Superpack)** - A standalone application for downloading Modrinth modpacks
 - **[Infinidoge/nix-minecraft](https://github.com/Infinidoge/nix-minecraft)** - An attempt to better package and support Minecraft as part of the Nix ecosystem
+- **[devBoi76/modrinthify](https://github.com/devBoi76/modrinthify)** - A browser extension to automatically redirect Curseforge projects to Modrinth whenever possible
```

Comment:
Alphabetical order please, and also capitalise CurseForge

Reply:
Done :)